sdk-base/src/main/java/com/mapbox/maps/plugin/gestures/generated/GesturesSettingsInterface.kt

```diff
@@ -38,6 +38,11 @@ interface GesturesSettingsInterface {
    */
   var scrollEnabled: Boolean
 
+  /**
+   * Whether rotation is enabled for the pinch to zoom gesture.
+   */
+  var simultaneousRotateAndPinchToZoomEnabled: Boolean
```

Comment:
Not a blocker - but it seems sooo a long name, do we want smth shorter like `pinchRotateEnabled` ?